### ISSUE TITLE
Uncaught TypeError during processing of AlertManeuver RPC

### ISSUE THREAD

**Issue by GetmanetsIrina** [smartdevicelink/sdl_hmi#404]
**Bug Report**
Uncaught TypeError during the processing of AlertManeuver RPC

**Preconditions:**
1. "Navigation-1" group is assigned to the app
2. HMI and SDL are started
3. Mobile app is registered and activated

**Reproduction Steps**
1. Mobile app requests AlertManeuver RPCeuver RPC

**Expected Behavior**
1. HMI processes the received requests and responds with SUCCESS resultCode

**Observed Behavior**
HMI does not respond to Navigation.AlertManeuver
```
AlertManeuverPopUp.js:181 Uncaught TypeError: this.set is not a function
    at AlertManeuverPopUp.js:181AlertManeuverPopUp.js:181 Uncaught TypeError: this.set is not a function
    at AlertManeuverPopUp.js:181
```

OS & Version Information
OS: Ubuntu 18.04
SDL: https://github.com/smartdevicelink/sdl_core/commit/a36ea82740e648debf1b0367c24fed85d6ef2d80
HMI: https://github.com/smartdevicelink/sdl_hmi/commit/6d32f1ba10bd6d2335d755d902788fa82c6286e9
Mobile app: SPT 20200831-Android

**Comment by LitvinenkoIra** [comment]
@theresalech Please note, this is a regression issue found by Luxoft during release manual testing.

**Comment by iCollin** [comment]
Closed via #404
